cloud-chatops/docker-compose.yaml

```diff
@@ -1,5 +1,5 @@
 services:
   cloud_chatops:
-    image: harbor.stfc.ac.uk/stfc-cloud/cloud-chatops:1.0.6
+    image: harbor.stfc.ac.uk/stfc-cloud/cloud-chatops:1.1.6
```

Comment:
Nit, according to SEMVER we should reset the patch version on a minor bump
https://semver.org/#spec-item-7

Reply:
Yes, my mistake